src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -47,6 +47,14 @@ internal sealed class ODataUtf8JsonWriter : IJsonWriter, IDisposable, IAsyncDisp
         private bool disposed;
         private const int chunkSize = 2048;
         /// <summary>
+        /// Content type of a value being written using TextWriter
+        /// </summary>
+        private string currentContentType;

```

Comment:
If it's only use from the text writer scope, maybe the naming could be more specific?
```suggestion
        private string textWriterCurrentContentType;

```